tmt/base.py

```diff
@@ -67,11 +67,12 @@
     cached_property,
     container_field,
     dict_to_yaml,
+    effective_workdir_root,
     field,
     git_clone,
     normalize_shell_script,
     verdict,
-    )
+)
```

Comment:
Two unrelated formatting changes, here and on line 26.

Reply:
updated^^